.all-contributorsrc

```diff
@@ -2145,6 +2145,10 @@
       "contributions": [
         "bug",
         "code",
+        "doc",
+        "ideas",
```

Comment:
you could also add `"mentoring"`, `"question"`

Reply:
Wasn't aware, will add while updating the test as you suggested below.